### PR TITLE
UI layout updates

### PR DESCRIPTION
**Pull request by ackernaut**
- Agenda list is a scrollable panel (improves both Meeting and Dash layout overall)
- Improved project column layout, including min-width
- Dash sidebar has Parabol branding

### CODE REVIEW

File: src/universal/modules/meeting/components/MeetingLayout/MeetingLayout.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ MeetingLayout.propTypes = {
 styles = StyleSheet.create({
   root: {
     backgroundColor: '#fff',
+    content: '"MeetingLayout"',
```
Comment (ackernaut) [inline src/universal/modules/meeting/components/MeetingLayout/MeetingLayout.js]: Hehe, was tracking down styles:component. Maybe I should be using a React DOM inspector? I can clean these up at any rate.

File: src/universal/routes/index.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ export default store => ({
     require('./landing'),
     require('./logout'),
     require('./meeting')(store),
-    ...require('./meetingLayoutRoutes'),
+    // ...require('./meetingLayoutRoutes'),
```
Comment (jordanh) [inline src/universal/routes/index.js]: Good catch

File: src/universal/styles/theme/images/brand/parabol-lockup-h.svg [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (jordanh) [inline src/universal/styles/theme/images/brand/parabol-lockup-h.svg]: Cool!

File: src/universal/styles/ui.js [hunk 4]
```diff
@@ -7,12 +7,14 @@ const ui = {
   actionCardBgColor: theme.palette.light60l,
   cardBorderColor: theme.palette.mid30l,
   cardBorderRadius: '.25rem',
-  cardPaddingBase: '.5rem',
   cardMinHeight: '7.5rem',
+  cardPaddingBase: '.5rem',
```
Comment (jordanh) [inline src/universal/styles/ui.js]: I think these are all obvious enough. As you add them, you may want to comment some of them for future posterity/maintainability.

**Comment by jordanh** [comment]
Wow @ackernaut. Lots of good stuff in here!

**Pull request merged**